Dockerfile

```diff
@@ -123,11 +148,24 @@ function fetch_and_build {
   fi
   
   echo "Fetching code from $1"
-  rm -rf /kernelcode/* /kernelcode/.* 2>/dev/null || true
+  rm -rf /home/nixuser/kernelcode/* /home/nixuser/kernelcode/.* 2>/dev/null || true
   git lfs install
-  git clone "$1" /kernelcode
-  cd /kernelcode
+  git clone "$1" /home/nixuser/kernelcode
+  cd /home/nixuser/kernelcode
   build_extension
+  echo "Build completed. Results are in /home/nixuser/kernelcode/build/"
+  
+  # skip login to huggingface since token is set in the env
+  # check user
+  nix shell nixpkgs#python3 nixpkgs#python3Packages.huggingface-hub -c huggingface-cli whoami
+
+  # upload the build to the repo
+  nix shell nixpkgs#python3 nixpkgs#python3Packages.huggingface-hub -c huggingface-cli \
+    upload \
+    --commit-message "Build from kernel-builder job" \
+    kernels-community/job-build-test-repo \
```

Comment:
This looks like testing code? Or at least, this would be different for every kernel?

Reply:
ah yea this is leftover from testing, its been removed in the latest changes and now pushes back into the repo

Reply:
I think I'm missing something, but this seems to upload unconditionally to the `kernels-community/job-build-test-repo` repo?

Reply:
🤦‍♂️🤦‍♂️ woops sorry about that, this is 100% a mistake when I removed the cmake changes I accidentally reverted the fix to this. 

fixed in the latest changes. tripled checked